trunk/source/CACSL2BoogieTranslator/src/de/uni_freiburg/informatik/ultimate/cdt/translation/implementation/base/chandler/BooleanArrayHelper_Bitvector.java

```diff
@@ -0,0 +1,37 @@
+package de.uni_freiburg.informatik.ultimate.cdt.translation.implementation.base.chandler;
+
+import de.uni_freiburg.informatik.ultimate.boogie.ExpressionFactory;
+import de.uni_freiburg.informatik.ultimate.boogie.ast.ASTType;
+import de.uni_freiburg.informatik.ultimate.boogie.ast.Expression;
+import de.uni_freiburg.informatik.ultimate.boogie.ast.PrimitiveType;
+import de.uni_freiburg.informatik.ultimate.boogie.ast.BinaryExpression.Operator;
+import de.uni_freiburg.informatik.ultimate.boogie.type.BoogieType;
+import de.uni_freiburg.informatik.ultimate.cdt.translation.implementation.LocationFactory;
+import de.uni_freiburg.informatik.ultimate.core.model.models.ILocation;
+
+public final class BooleanArrayHelper_Bitvector implements IBooleanArrayHelper {
```

Comment:
Underscores in class names should be avoided according to the Java camel case style. This also applies to the name of this source code file.